<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>蜡烛图，时间日期坐标轴</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab_Data">
        <attribute name="title">
         <string>数据</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>50</number>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>24</number>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_Setup">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <attribute name="title">
         <string>图表设置</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frameData">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>图表</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="verticalSpacing">
                <number>10</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>图表主题</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboTheme">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Light</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BlueCerulean</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Dark</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BrownSand</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BlueNcs</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>HighContrast</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BlueIcy</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Qt</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="chkBox_Legend">
                 <property name="text">
                  <string>显示图例</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>蜡烛图</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QCheckBox" name="chkBox_Caps">
                 <property name="text">
                  <string>capsVisible</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkBox_Outline">
                 <property name="text">
                  <string>bodyOutlineVisible</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Y轴--QValueAxis</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="verticalSpacing">
                <number>10</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>最小值</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="spinY_Min">
                 <property name="maximum">
                  <double>1000000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="btnY_SetRange">
                 <property name="text">
                  <string>设置范围</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>最大值</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="spinY_Max">
                 <property name="maximum">
                  <double>1000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>20.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>分度个数</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="spinY_Ticks">
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>X轴--QDateTimeAxis</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <property name="verticalSpacing">
                <number>10</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>分度个数</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="comboDateFormat">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>MM-dd</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>M-d</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>M月d日</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>yyyy-MM-dd</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>日期格式</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="btnX_Ticks">
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_chart">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QmyChartView" name="chartView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomReset"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actZoomReset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/414.bmp</normaloff>:/images/images/414.bmp</iconset>
   </property>
   <property name="text">
    <string>原始大小</string>
   </property>
   <property name="toolTip">
    <string>原始大小</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/418.bmp</normaloff>:/images/images/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/416.bmp</normaloff>:/images/images/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/001.GIF</normaloff>:/images/images/001.GIF</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开数据文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmyChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">myChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
